Documentation/cli-installation-scenarios.md

```diff
@@ -0,0 +1,214 @@
+Obtaining .NET CLI 
+==================
+
+# Contents
```

Comment:
Might be worth dropping these to a `h2`? They're awfully big right now, and `Obtaining .NET CLI` is `h1` :smile: